tasks.sh

```diff
@@ -216,8 +216,12 @@ case "${COMMAND}" in
           --create-with-perms=0644,ud+X:gd-rwX:od-rwX ${img_mnt} ${img_bind_mnt}
 
       # Debian rootfs generation and config setting
-      sudo mmdebstrap --include ssh,acpid,acpi-support-base,gdb,systemtap,file,psmisc,strace,vim,bpftool,bpftrace,trace-cmd,linux-perf \
-          --arch ${DEBIAN_TARGET_ARCH} unstable ${img_mnt}
+      sudo mkosi --package=ssh,acpid,acpi-support-base,gdb,systemtap,file,psmisc,strace,vim,bpftool,bpftrace,trace-cmd,linux-perf \
+      --architecture=${DEBIAN_TARGET_ARCH} --distribution=debian --release=unstable --output-dir=${img_mnt} --format=directory
+
+      # Move mkosi-generated rootfs from ${img_mnt}/image to ${img_mnt} to match script's expected directory structure
+      sudo mv ${img_mnt}/image/* ${img_mnt} && sudo rmdir ${img_mnt}/image
+
       echo "debian-vm" > ${img_bind_mnt}/etc/hostname
       echo "nameserver 8.8.8.8" > ${img_bind_mnt}/etc/resolv.conf
       echo "hostfs /host 9p trans=virtio,rw,nofail 0 0" > ${img_bind_mnt}/etc/fstab
```

Comment:
I wonder if the rest of this manual step should be instead done with a `--extra-tree` or some `--postinst-script`. I would very much like to keep everything self contained in `tasks.sh` but maybe we could output these files to some dumb tmp directory and then feed it to mkosi. My understanding is that mkosi should then be able to add the files to the target and we could drop the mount and bind mount operations we're doing now and most importantly drop the need for root (it's awkward having to sudo to do some kernel development).

I wonder if  the mkosi `--ssh true` flag could also help us drop the `SSH_KEY` dance